frontend/src/modules/_shared/components/ColorLegendsContainer/colorLegendsContainer.tsx

```diff
@@ -4,23 +4,26 @@ import { ColorScale, ColorScaleGradientType, ColorScaleType } from "@lib/utils/C
 import { resolveClassNames } from "@lib/utils/resolveClassNames";
 import { ColorScaleWithName } from "@modules_shared/utils/ColorScaleWithName";
 
-const lineWidth = 6;
-const lineColor = "#555";
-const textGap = 6;
-const offset = 10;
-const legendGap = 4;
-const textWidth = 70;
-const nameLabelWidth = 10;
-const fontSize = 10;
-
-const textStyle: React.CSSProperties = {
+const PREFERENCES = {
```

Comment:
Is "Internal Style" or something similar a better name? As `PREFERENCES` is not really a preference/opt-in, but actually a style added in addition to the `ColorLegendProp` provided to the component

Reply:
Going with "STYLE_CONSTANTS".